lib/litle/api.rb

```diff
@@ -4,9 +4,10 @@ class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
       def initialize
         gateway_builder = Proc.new do |config|
-          ::ActiveMerchant::Billing::LitleGateway.new :login => config[:login],
+          ::ActiveMerchant::Billing::LitleGateway.new :login => config[:username],
```

Comment:
Looks like a typo in the file itself, see https://github.com/killbill/killbill-litle-plugin/commit/f5bbfd52b8d657a72267f2d782df37b750649b1a#diff-2fe9adc42378a48d1656caeb98453f11 (seems better to keep `login`).